projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4952,6 +4998,25 @@ public static function authorize_ending_authorized( $data ) {
 		self::handle_post_authorization_actions( $activate_sso, $do_redirect_on_error );
 	}
 
+	/**
+	 * Fires on the jetpack_site_registered hook and acitvates default modules
+	 */
+	public static function site_registered() {
```

Comment:
Can we rename this to something more contextual like `activate_default_registered_modules`?

Reply:
done in https://github.com/Automattic/jetpack/pull/19068/commits/81863a0730e0607988093b37c402a344ac26860b